h/static/scripts/app-controller.coffee

```diff
@@ -30,15 +32,27 @@ module.exports = class AppController
     $scope.accountDialog = visible: false
     $scope.shareDialog = visible: false
 
-    # Check to see if we are on the stream page so we can hide share button.
-    $scope.isEmbedded = $window.top isnt $window
+    # Check to see if we're in the sidebar, or on a standalone page such as
+    # the stream page or an individual annotation page.
+    $scope.isSidebar = $window.top isnt $window
 
     # Default sort
     $scope.sort = name: 'Location'
 
+    $scope.$on('groupFocused', (event) ->
```

Comment:
I don't think the approach of using events is bad, but I do think it would help to have all the events that are used across the application defined in one place. They can then be referenced elsewhere.

eg:

```js
// event-types.js
module.exports = {
  /** Description of when event is used and the associated payload data */
  GROUP_FOCUSED: 'groupFocused'
};

// some-other-file.js
var events = require('path/to/event-types');

$scope.$on(events.GROUP_FOCUSED, function (event) {
 ...
});
```